monitoring/lifecycle/dashboard.py

```diff
@@ -271,7 +271,7 @@ def kafka_row(topic, op):
             layout.row([lifecycle_global_s3_requests], height=10),
             layout.row(lifecycle_global_s3_error_rates, height=4),
```

Comment:
dashboard will be updated in a separate PR ?

Reply:
😕
this breaks the kind of "atomicity" we get from having the dashboards in the same repo as the code (and thus the safety of having both the dashboard and matching metrics in the product)